.ci/azure/linux_arm64.yml

```diff
@@ -0,0 +1,134 @@
+resources:
+  repositories:
+  - repository: openvino
+    type: github
+    endpoint: openvinotoolkit
+    name: openvinotoolkit/openvino
+
+jobs:
+- job: Lin
+  # About 150% of total time
+  timeoutInMinutes: 60
+
+  pool:
+    name: LIN_VMSS_VENV_F16S_WU2
+
+  variables:
+    system.debug: true
+    VSTS_HTTP_RETRY: 5
+    VSTS_HTTP_TIMEOUT: 200
+    WORKERS_NUMBER: 16
+    BUILD_TYPE: Release
+    REPO_DIR: $(Build.Repository.LocalPath)
+    OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
+    OPENCV_REPO_DIR: $(REPO_DIR)/../opencv
+    WORK_DIR: $(Pipeline.Workspace)/_w
+    BUILD_DIR: $(WORK_DIR)/build
+    BUILD_DIR_ARM: $(WORK_DIR)/build_arm
+    BUILD_DIR_OPENCV: $(WORK_DIR)/build_opencv
+
+  steps:
+  - script: |
+      curl -H Metadata:true --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2019-06-01"
+      whoami
+      uname -a
+      echo Python3 info ; which python3 ; python3 --version
+      echo Python info ; which python ; python --version
+      echo gcc info ; which gcc ; gcc --version
+      lsb_release
+      env
+      cat /proc/cpuinfo
+      cat /proc/meminfo
+      cat /etc/fstab
+      vmstat -s
+      df
+      lsblk -o NAME,HCTL,SIZE,MOUNTPOINT | grep -i "sd"
+      free -h
+    displayName: 'System info'
+
+  - script: |
+      rm -rf $(WORK_DIR) ; mkdir $(WORK_DIR)
+      rm -rf $(BUILD_DIR) ; mkdir $(BUILD_DIR)
+    displayName: 'Make dir'
+
+  - checkout: self
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino_contrib
+
+  - checkout: openvino
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino
+
+  - script: |
+      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev protobuf-compiler
+      # Speed up build
+      wget https://github.com/ninja-build/ninja/releases/download/v1.10.0/ninja-linux.zip
+      unzip ninja-linux.zip
+      sudo cp -v ninja /usr/local/bin/
+      python3 -m pip install --upgrade pip
+      python3 -m pip install cython
+      git clone https://github.com/opencv/opencv.git --depth 1 $(OPENCV_REPO_DIR)
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Install dependencies'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DBUILD_LIST=imgcodecs,videoio,highgui
+        -DCMAKE_TOOLCHAIN_FILE=$(OPENCV_REPO_DIR)/platforms/linux/aarch64-gnu.toolchain.cmake
+        $(OPENCV_REPO_DIR)
+      workingDirectory: $(BUILD_DIR_OPENCV)
+
+  - script: |
+      ninja
+      ninja install
+    workingDirectory: $(BUILD_DIR_OPENCV)
+    displayName: 'Build OpenCV Lin ARM64'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DCMAKE_TOOLCHAIN_FILE=$(OPENVINO_REPO_DIR)/cmake/arm64.toolchain.cmake
+        -DOpenCV_DIR=$(BUILD_DIR_OPENCV)/install/lib/cmake/opencv4
+        -DENABLE_OPENCV=OFF
```

Comment:
why OPENCV is OFF while provide path to OpenCV?

Reply:
It's a workaround to disable automatic OpenCV package downloading. Copied from https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/Dockerfile.RPi64#L47